libs/components/theme/src/lib/styles/_public-api/themes/modern/_compat/_mixins.scss

```diff
@@ -54,8 +54,7 @@
 
 @mixin sky-theme-modern-btn-disabled($color, $background, $border) {
   &.sky-btn-disabled,
-  &[disabled],
-  fieldset[disabled] & {
```

Comment:
This rule was not working at all and was causing issues when used inside the `host-context` of the modern theme mixin. Removing and will file a bug.